pkg/trace/config/subscriber.go

```diff
@@ -31,6 +32,19 @@ func (s *Subscriber) Get(req *pbgo.GetConfigsRequest) (*pbgo.ConfigResponse, err
 		return nil, errors.New("not allowed")
 	}
 	s.once.Do(s.subscribe)
+LOOP:
+	for {
+		select {
+		case s.tracerInfos <- req.TracerInfo:
+			break LOOP
+		default:
+			select {
+			case <-s.tracerInfos:
+				log.Warnf("Cannot add more tracers. Dropping request.")
+			default: // no-op
+			}
```

Comment:
```suggestion
			default:
				// no-op
			}
```
Can we please put it on the next line?